python/taichi/lang/kernel_arguments.py

```diff
@@ -9,6 +9,12 @@
 from taichi.types.primitive_types import u64
 
 
+class KernelArguments:
```

Comment:
nit: Here and below, please use `KernelArgument` instead, because it represents a single argument.

```suggestion
class KernelArgument:
```

Reply:
Sry for that. I should have fixed it in the last pr. Thanks for your time.